sdk/tests/__init__.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date, datetime
+from turing import generated as client
+
+
+def json_serializer(o):
+    if isinstance(o, (date, datetime)):
+        return o.isoformat()
+    if isinstance(o, client.model_utils.ModelNormal):
+        return o.to_dict()
+
+
+project_1 = client.models.Project(
```

Comment:
Curious - is this common practice to define test data in the `__init__.py` rather than pytest fixtures ?

Reply:
It's wild wild west, everyone is doing it in their own way :)

but on a serious note, I didn't want to make it as individual fixtures (such as `project_1`, `project_2`...) because then they will have to be included into the test method's argument list and it could potentially grow arguments list unbearable long. 
I'll re-evaluate what can be made as a fixture after I'm done with all the tests

Reply:
This is addressed now too